vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/AbsMessageItem.kt

```diff
@@ -90,7 +91,12 @@ abstract class AbsMessageItem<H : AbsMessageItem.Holder> : AbsBaseMessageItem<H>
         }
         if (attributes.informationData.messageLayout.showDisplayName) {
             holder.memberNameView.isVisible = true
-            holder.memberNameView.text = attributes.informationData.memberName
+            // Tchap: Remove domain name in case of DM.
+            holder.memberNameView.text = if (attributes.isDirect != null && attributes.isDirect == true) {
```

Comment:
the nullability check seems useless as it will also be checked in second part of the condition (equals to true means it is not null)